### DISCUSSION TITLE
Best way to learn the Vits model

### DISCUSSION THREAD

**Discussion by harmlessman** [coqui-ai/TTS#1932] [General Q&A] [answered]
My goal is to make a model that is fluent in two languages.
I have 12 hours of Korean data, 5+5 hours of Japanese data, and 1 hour of target data. The target data is a little under-quality, but the other data is the best quality.

When I first started learning, I learned all of the above data at once. After going up to 15k, I synthesized the voice and heard the voice, and it was enough to understand that it was the voice of the target data, but the voice was broken in a particular sound and it wasn't satisfactory. (0.6.2 version)

I read the answers from Discussions one by one and tried to find a solution, but I think it would be most accurate to ask this question.
The model used is Vits.

1. Learning the above data at once, learning other than the target data first, and then inserting and learning the target data later, which is better for good quality? (fine tune)

2. There is something called speaker embedding, would it help you to do this?

3. How many hours of data do you need for a quality similar to that of a human voice in Vits? I'm asking because I don't know how much data is needed. (base data, target data)

I'd appreciate it if you could reply.

**Comment by erogol** [comment, marked as answer]
> Learning the above data at once, learning other than the target data first, and then inserting and learning the target data later, which is better for good quality? (fine tune)

Start with the good dataset and then add the remaining to fine-tune

> There is something called speaker embedding, would it help you to do this?

If you train multi-speaker dataset yes you need speaker embeddings

> How many hours of data do you need for a quality similar to that of a human voice in Vits? I'm asking because I don't know how much data is needed. (base data, target data)

Check https://tts.readthedocs.io/en/latest/what_makes_a_good_dataset.html

**Reply by harmlessman** [reply]
Thank you for your answer!
I will run syntheis based on your answer!

**Reply by huongtxvega** [reply]
@harmlessman Hi, Have you tried it yet?

**Reply by harmlessman** [reply]
The training was carried out in the above method, and good results were obtained. However, some noise occurred during synthesis, perhaps because of the noise in the target data.
